pdlua_gfx.h

```diff
@@ -915,6 +923,7 @@ static int end_paint(lua_State* L) {
     int xpos = text_xpix((t_object*)obj, obj->canvas);
     int ypos = text_ypix((t_object*)obj, obj->canvas);
 
+    // TODO: I don't think we need to call drawiofor on each layer?
```

Comment:
I think that as long as the paint method can change the size of the object no matter what the layer is, we need to keep this for all layers, I'm afraid. However, we might be able to optimize this by recording the size in start_paint and then check in end_paint whether the size was changed.

Reply:
The set_size method can be called from anywhere, not just the paint functions. So yeah, I think we need to keep track of it.

Reply:
Yeah, let's just keep this in mind for for future optimization opportunities. Rome wasn't build in a day either.